web/packages/teleport/src/Audit/EventList/EventTypeCell.tsx

```diff
@@ -225,12 +225,11 @@ const EventIconMap: Record<EventCode, React.FC> = {
 };
 
 export default function renderTypeCell(event: Event, clusterId: string) {
-  const IconType = EventIconMap[event.code] || Icons.List;
+  const Icon = EventIconMap[event.code] || Icons.ListThin;
 
   const iconProps = {
-    p: '1',
-    mr: '3',
-    fontSize: '3',
```

Comment:
the audit log screen has icons that are more bigger. it's too big imo, it increases vertical spacing and decreases width for some column.

take a look, icons are not the most important thing to display, so i think we can decrease the size? i'm using macbook pro 16 inch full screen

<img width="1412" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/cb1fb6e4-8d29-467f-baaf-4c08d37d0479">